### PR TITLE
Fix musl

### PR DESCRIPTION
**Pull request by polachok**

### CODE REVIEW

**Comment by japaric** [comment]
> unrecognized relocation (0x2b) in section `.text.malloc_conf_init'

this was a problem with how upstream binaries were built. It should fix itself by next stable release.

Regardless I think it is helpful to install the musl-packages that this PR proposes.

File: ci/install.sh [hunk 1]
```diff
@@ -10,6 +10,9 @@ install_c_toolchain() {
             sudo apt-get install -y --no-install-recommends \
                  gcc-aarch64-linux-gnu libc6-arm64-cross libc6-dev-arm64-cross
             ;;
+	*-musl)
+	    sudo apt-get install musl-tools
```
Comment (japaric) [inline ci/install.sh]: this shouldn't be needed because musl_packages already includes it, right?

Comment (polachok) [reply ci/install.sh]: It seems like install.sh is called before musl_packages, see log:
https://travis-ci.org/japaric/rust-everywhere/jobs/163442305

Comment (japaric) [reply ci/install.sh]: Hmm, no. I'm certain that install.sh runs after the apt-get packages get installed. Instead it seems that no apt-get packages are installed for the stable i686-musl target; I don't know why though. There's no "Installing APT Packages (BETA)" at the top. See

[x86_64](https://travis-ci.org/japaric/rust-everywhere/jobs/163442310#L237) vs
[i686](https://travis-ci.org/japaric/rust-everywhere/jobs/163442305#L237)

Comment (polachok) [reply ci/install.sh]: Seems like i fixed that with my last commit (bad indent)

**Comment by japaric** [comment]
Thank you @polachok!

The other errors are due to rust-lang/rust#36793 and should be fixed by next nightly. This fix will probably be backported to the beta so it should also be fixed there soon-ish.

**Pull request merged**